src/server/scripts/World/go_scripts.cpp

```diff
@@ -1898,22 +1898,25 @@ class go_pirate_day_music : public GameObjectScript
 
 enum BellHourlySoundFX
 {
-    BELLTOLLHORDE          = 6595, // Horde
-    BELLTOLLTRIBAL         = 6675,
-    BELLTOLLALLIANCE       = 6594, // Alliance
-    BELLTOLLNIGHTELF       = 6674,
-    BELLTOLLDWARFGNOME     = 7234,
-    BELLTOLLKHARAZHAN      = 9154 // Kharazhan
+    BELLTOLLHORDE      = 6595, // Undercity
+    BELLTOLLTRIBAL     = 6675, // Orgrimma/Thunderbluff
+    BELLTOLLALLIANCE   = 6594, // Stormwind
+    BELLTOLLNIGHTELF   = 6674, // Darnassus
+    BELLTOLLDWARFGNOME = 7234, // Ironforge
+    BELLTOLLKHARAZHAN  = 9154  // Kharazhan
 };
 
-enum BellHourlySoundAreas
+enum BellHourlySoundZones
 {
-    UNDERCITY_AREA         = 1497,
-    IRONFORGE_1_AREA       = 809,
-    IRONFORGE_2_AREA       = 1,
-    DARNASSUS_AREA         = 1657,
-    TELDRASSIL_ZONE        = 141,
-    KHARAZHAN_MAPID        = 532
+    TIRISFAL_ZONE       = 85,
+    UNDERCITY_ZONE      = 1497,
+    DUN_MOROGH_ZONE     = 1,
+    IRONFORGE_ZONE      = 1537,
+    TELDRASSIL_ZONE     = 141,
+    DARNASSUS_ZONE      = 1657,
+    ASHENVALE_ZONE      = 331,
+    HILLSBARD_FOOTHILLS = 267,
```

Comment:
typo (bard => brad):
```suggestion
    HILLSBRAD_FOOTHILLS = 267,
```